gcsfs/core.py

```diff
@@ -247,6 +247,7 @@ def __init__(
         loop=None,
         timeout=None,
         endpoint_url=None,
+        location="US",
```

Comment:
I would suggest defaulting to `None`, and only adding it to the json args if it is not `None`. I say this, because the default location may well depend on the user's location and/or defaults in their account.

Reply:
that's a good idea. i'll go ahead and make the change.